content/en/Getting started/glossary.md

```diff
@@ -86,11 +86,13 @@ In the context of a Cobalt pentest, you can specify one of three options for an
 ## Finding
```

Comment:
Git isn't letting me put a comment higher than this. Just noticed that the CIS acronym doesn't line up with how we present the other acronyms here. Should probably be Center for Internet Security (CIS) to match the OWASP and others below.

Reply:
Quite right. Fixing.